hw/ip/otp_ctrl/doc/otp_ctrl_digests.md

```diff
@@ -0,0 +1,8 @@
+|                      Digest Name                      |   Affected Partition  |  Calculated by HW  |
```

Comment:
since these are generated, does it make sense to have a comment header warning the users not to modify?

Reply:
good point. however it needs to be an HTML comment, right? otherwise markdown would probably render it...

Reply:
ah... does it need to be rendered? i was just thinking some text that doesn't cause any breakage that warns the user off.

Reply:
yeah hugo translates the MD table into an HTML table. I've added a HTML comment, that works.